modules/cnv-refreshing-certificates.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-refresh-certificates.adoc
+
+[id="cnv-refreshing-certificates_{context}"]
+= Refreshing TLS certificates
+
+To refresh the TLS certificates for {CNVProductName}, download and execute the `rotate-certs` script. This script is available from the `kubevirt/hyperconverged-cluster-operator` repository on GitHub. 
```

Comment:
Question: are there any issues with including this GitHub info in the downstream docs?

Reply:
This was at the request of PM team. AFAIK it will be fixed in 2.3